apps/api/src/music/music.service.ts

```diff
@@ -67,13 +74,35 @@ export class MusicService {
     }
   }
 
+  private purgeAssociatedFiles(music: Music) {
+    const audioFilePath =
+      this.configService.get(ENV_NAME.STORAGE_DEST) +
+      '/musics/' +
+      music.musicAudioFileName;
+    const coverFilePath =
+      this.configService.get(ENV_NAME.STORAGE_DEST) +
+      '/covers/' +
+      music.coverImageFileName;
+    fs.unlink(audioFilePath, (err) => {
+      if (err) {
+        throw err;
+      }
+      console.log('File is deleted.');
+    });
+    fs.unlink(coverFilePath, (err) => {
+      if (err) {
+        throw err;
+      }
+      console.log('File is deleted.');
+    });
+  }
```

Comment:
we could extract the whole file management into a separate service/moudle. #SOLID

Reply:
Yup, LGTM

Reply:
Follow up on #20